spec/features/neighborhoods/user_searches_by_neighborhood_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+feature 'user can search by neighborhood' do
+
+  scenario 'visitor searches for a neighborhood successfully' do
+    neighborhood_1 = FactoryGirl.create(:neighborhood, name: "One Neighborhood")
+    neighborhood_2 = FactoryGirl.create(:neighborhood, name: "Two Neighborhood")
+
+    visit neighborhoods_path
+
+    fill_in 'Search Neighborhoods', with: "one"
+    click_on "Search"
+
+    expect(page).to have_content("One Neighborhood")
+    expect(page).to_not have_content("Two Neighborhood")
+  end
+
+  scenario 'visitor searches for something that does not exist' do
+    neighborhood_1 = FactoryGirl.create(:neighborhood, name: "One Neighborhood")
+    neighborhood_2 = FactoryGirl.create(:neighborhood, name: "Two Neighborhood")
```

Comment:
Useless assignment to variable - `neighborhood_2`. Did you mean `neighborhood_1`?